python/taichi/lang/ast/ast_transformer.py

```diff
@@ -1060,7 +1060,7 @@ def build_static_for(ctx, node, is_grouped):
                     f"Group for should have 1 loop target, found {len(targets)}"
                 )
             target = targets[0]
-            for value in impl.grouped(ndrange_arg):
+            for value in impl.static(impl.grouped(ndrange_arg)):
```

Comment:
```suggestion
            for value in impl.grouped(ndrange_arg):
```
Now we don't need to call `static` in Python.